src/deno_json/mod.rs

```diff
@@ -453,18 +453,25 @@ pub struct WorkspaceConfig {
 #[error("Failed to parse \"patch\" configuration.")]
 pub struct PatchConfigParseError(#[source] serde_json::Error);
 
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct TaskDefinition {
+  pub command: String,
+  #[serde(default)]
+  pub dependencies: Vec<String>,
+}
+
 #[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 #[serde(untagged)]
 pub enum Task {
-  Definition(String),
+  Definition(TaskDefinition),
   Commented {
```

Comment:
While we're at it, I think we maybe should consider dropping support for jsdocs for tasks? We can add `description` field to the object notation instead. I don't think we ever documented it besides the blog post